test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -4036,7 +4036,7 @@ BOOST_AUTO_TEST_CASE(varM_disqualified_as_keyword)
 			}
 		}
 	)";
-	CHECK_ERROR(text, DeclarationError, "Identifier not found or not unique.");
+	CHECK_ERROR_ALLOW_MULTI(text, DeclarationError, "Identifier not found or not unique.");
```

Comment:
Strange there's only one test affected by the changes. Any chance to increase our test coverage for the rest of the changes?

Reply:
What tests would you suggest adding?

Reply:
The tests I just added should cover most cases.